dnnlib/comet_utils.py

```diff
@@ -0,0 +1,19 @@
+from comet_ml import Experiment
+import tensorflow as tf
+import json
+
+class CometLogger():
+    def __init__(self,comet_config):
+        with open(comet_config, "r") as content:
+            content=json.load(content)
+            self.experiment = Experiment(content['api_key'],content['project_name'],content['workspace'])
```

Comment:
```suggestion
    def __init__(self):
            self.experiment = Experiment()
```

Reply:
comet_ml already has methods for setting api_key, project_name, and workspace through config files, or environment variables. I think it would be a problem for most users to have to duplicate this in another config format.